scripts/trigger-action

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# e - non-interactive terminals will exit on failures immediately
+set -e
+
+usage() {
+  echo "usage: $(basename "$0") <event_type> [<client_payload>]"
+  echo
+  echo "    A GitHub personal access token with \"Repo\" access must be"
+  echo "    availabe in the environment variable GITHUB_TOKEN."
+  echo
+}
+
+if [ "$#" -lt 1 ]; then
+  usage >&2
+  exit 1
+fi
+
+curl -X POST https://api.github.com/repos/tenzir/vast/dispatches \
```

Comment:
Maybe we should change the script to take the target repository (i.e. `tenzir/vast` or `tenzir/event-horizon`) as first argument? Then we can use the same script for everything.

Reply:
That can probably be extracted from `git remote -v`, but I don't want to get side tracked by this now.